tools/db_bench_tool.cc

```diff
@@ -2474,6 +2488,11 @@ void VerifyDBFromDB(std::string& truth_db_name) {
         }
       }
 
+      if (FLAGS_use_keep_filter && FLAGS_expire_style != "compaction_filter") {
+        filter.reset(new KeepFilter());
+        fprintf(stdout, "A noop compaction filter is used\n");
+        open_options_.compaction_filter = filter.get();
```

Comment:
I think this belongs in `InitializeOptionsGeneral`. The pattern you followed probably did it in this function because they were making an option that is only set for a specific benchmark; however, the option you're adding is general to all benchmarks.